README.md

```diff
@@ -32,8 +32,7 @@ pos-cli init --url https://github.com/mdyd-dev/marketplace-template-poc.git
 - build assets
 
 ```sh
-npm ci
-npm build
+npm i && npm run build
```

Comment:
```
npm install
npm run build
```

Might be easier to understand for newbies :)

Reply:
Thank you, edited.